RealmBrowser/Application/RLMApplicationDelegate.m

```diff
@@ -178,6 +180,23 @@ - (IBAction)showWelcomeWindow:(id)sender {
     }];
 
     [self addAuxiliaryWindowController:welcomeWindowController];
+    if (!toutDisplayedDuringAppRun && ![[NSUserDefaults standardUserDefaults] boolForKey:toutSuppressionKey]) {
+        toutDisplayedDuringAppRun = YES;
+        NSAlert *upgradeAlert = [[NSAlert alloc] init];
+        [upgradeAlert addButtonWithTitle:@"Get Realm Studio..."];
+        [upgradeAlert addButtonWithTitle:@"Not now"];
+        [upgradeAlert setShowsSuppressionButton:YES];
+        [upgradeAlert setMessageText:@"Realm Browser is deprecated"];
+        [upgradeAlert setInformativeText:@"Realm Browser has been replaced by Realm Studio. Bug fixes and enhancements will only be available via Realm Studio going forwards."];
+        NSInteger result = [upgradeAlert runModal];
+        if (result == NSAlertFirstButtonReturn) {
+            // Open the web site.
+            [[NSWorkspace sharedWorkspace] openURL:[NSURL URLWithString:@"https://realm.io/products/realm-studio/"]];
```

Comment:
Since we know people are on a Mac, we could consider to just download the Studio for Mac directly. But this is fine as well.

Reply:
I think it makes sense for users to read up about Realm Studio and find out what they're downloading. If we just pop up a download file window without more context they may get suspicious.